### PR TITLE
Implements interfaces.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

- [x] ConnectionInterface.
- [x] ConnectionPDOInterface.
- [x] CommandInterface.
- [x] QuoterInterface.
- [x] SchemaInterface.
- [ ] QueryBuilderInterface.
- [x] TransactionInterface.
- [ ] DDLCommand.
- [ ] DQLCommand.
- [ ] DSLCommand.  
- [ ] PDOStatment.

### CODE REVIEW

File: src/Command/Command.php [hunk 1]
```diff
@@ -434,817 +284,365 @@ public function bindValues(array $values): self
...
-     */
-    public function dropCheck(string $name, string $table): self
+    public function queryScalar(): bool|string|null|int
     {
-        $sql = $this->db->getQueryBuilder()->dropCheck($name, $table);
-        return $this->setSql($sql)->requireTableSchemaRefresh($table);
+        $result = $this->queryInternal('fetchColumn', 0);
+
+        if (is_resource($result) && get_resource_type($result) === 'stream') {
+            return stream_get_contents($result);
```
Comment (darkdef) [inline src/Command/Command.php:544]: We cannot know anything about the return type of the values. Let's remove this implementation.

File: src/Command/Command.php [hunk 2]
```diff
@@ -434,817 +284,365 @@ public function bindValues(array $values): self
...
-     */
-    public function addCommentOnColumn(string $table, string $column, string $comment): self
+    public function setRawSql(string $sql): self
     {
-        $sql = $this->db->getQueryBuilder()->addCommentOnColumn($table, $column, $comment);
-        return $this->setSql($sql)->requireTableSchemaRefresh($table);
-    }
+        if ($sql !== $this->sql) {
+            $this->cancel();
+            $this->reset();
```
Comment (darkdef) [inline src/Command/Command.php:585]: it's details (need remove this from abstract class)

File: src/Command/Command.php [hunk 3]
```diff
@@ -434,817 +284,365 @@ public function bindValues(array $values): self
...
-     *
-     * @throws Throwable
-     * @throws Exception execution failed.
+     * Binds pending parameters that were registered via {@see bindValue()} and {@see bindValues()}.
      *
-     * @return int number of rows affected by the execution.
+     * Note that this method requires an active {@see pdoStatement}.
      */
-    public function execute(): int
+    protected function bindPendingParams(): void
```
Comment (darkdef) [inline src/Command/Command.php:628]: need remove and use only bindParam and bindParams functions

File: src/Command/Command.php [hunk 4]
```diff
@@ -79,263 +83,155 @@
...
-    /**
-     * Returns the SQL statement for this command.
-     *
-     * @return string|null the SQL statement to be executed.
-     */
-    public function getSql(): ?string
+    public function addColumn(string $table, string $column, string $type): self
     {
-        return $this->sql;
+        $sql = $this->queryBuilder->addColumn($table, $column, $type);
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply src/Command/Command.php]: fixed.

**Pull request merged**